src/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
 
   <!-- Build configuration -->
   <PropertyGroup>
+    <VersionPrefix>5.0.0</VersionPrefix>
```

Comment:
I'd consider putting this in the root Directory.Build.props instead of in src, for better visibility/clarity (even if the tests don't need it).

Reply:
I think that it make sense to put this line in the root directory, but change it later to have a separate version for each project in `src`.